docs/reference/target-group-policy.md

```diff
@@ -53,7 +55,15 @@ Updates to this configuration result in a replacement of VPC Lattice TargetGroup
 
 Appears on: TargetGroupPolicySpec
 
-HealthCheckConfig defines health check configuration for given VPC Lattice target group. For the detailed explanation and supported values, please refer to [VPC Lattice documentation](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) on health checks.
+HealthCheckConfig defines health check configuration for given VPC Lattice target group.
+
+For the detailed explanation and supported values, please refer to [VPC Lattice documentation](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) on health checks.
+
+**Limitations and Considerations**
+* Omitting `healthCheck` results in [VPC Lattice default behavior](https://docs.aws.amazon.com/vpc-lattice/latest/ug/target-group-health-checks.html) depending on the protocol.
+  * Health check is enabled by default for HTTP1 target groups.
+  * Health check is disabled by default for HTTP2/gRPC target groups.
+* For targets behind GRPCRoute, you should create a separate endpoint to enable health checks - HTTP/2 health check directly on gRPC endpoints is not supported.
```

Comment:
What this mean? ` you should create a separate endpoint to enable health checks`, other part LGTM

Reply:
slightly changed the wording there. It means you cannot directly perform HC on the target, so you need to enable another endpoint that is dedicated for health check.